### PR TITLE
fix logging and avoid replace for textile repo

### PR DESCRIPTION
**Pull request by jsign**
Trying to make `hubd` have working saving logs also to a file, went into a deep rabbit hole.

`hubd` uses an util function on `go-threads` to setup logging. This `SetupLogging` function used env vars to configure the file path output. At some point when we updated `go-log` from `v1.0.0/v1.0.1` to `v1.0.4`, and then something bad happened.

After `ipfs/go-log@v1.0.1` there is a [commit](https://github.com/ipfs/go-log/commit/e1c5faa6442f373b952970b80cb97ec09fd694d5#diff-17a9f1db2982c66c910a1e06f92e6489L34) that from `v1.0.2` and forward makes the `GOLOG_FILE` style of configuring the output file log break the way we are using it.

In that change, they use `go-log/v2` as the underlying logging library, which is OK. But `go-log/v2` only recognizes `GOLOG_FILE` [at `init()` time](https://github.com/ipfs/go-log/blob/0016c0b4b3e4f13beb8700f91d56af9dbf91aa54/setup.go#L16). The underlying "old" `loggingv1.SetupLogging()` doesn't re-setup things to take this into consideration. So changing the env var after program bootstrap (after `init()` calls), doesn't have the intended effect.

In short, I fixed our setup function to call again the same function that gets called in `init()` but with explicit args. The previous API of go-log which used env vars backfired.

If the above is confusing, don't worry... it is.

Apart from that, I updated `go-libp2p-peerstore` to `v0.2.6`. This new tag already includes a commit I made to fix a race condition that we solved with a `replace` in [textile go.mod](https://github.com/textileio/textile/blob/0f87b8f7165919941ba0d132885132d0a124a19c/go.mod#L83). So, whenever we update `textile` to a new `go-threads` tag that includes this PR, we could avoid that replace.

### CODE REVIEW

File: util/util.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/alecthomas/jsonschema"
 	ipfslite "github.com/hsanjuan/ipfs-lite"
 	logging "github.com/ipfs/go-log"
```
[the remark this answers is not in the document]
Comment (jsign) [reply util/util.go]: It is. Decided dimissing since we'd eventually use a better way to save long-term logs.

**Comment by jsign** [comment]
The `go-libp2p-peerstore` update appeared for free in the rebase.
